homeassistant/components/gios/config_flow.py

```diff
@@ -25,7 +24,7 @@
 class GiosFlowHandler(ConfigFlow, domain=DOMAIN):
     """Config flow for GIOS."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
As we discussed, I don't like the idea of ​​incrementing the version for such a minor change and preventing users from downgrading if they need to. I'd try increasing only the minor_version.